homeassistant/components/digital_ocean/constants.py

```diff
@@ -0,0 +1,12 @@
+"""Store constants used across the integration."""
+
+from datetime import timedelta
+
+HASS_DATA_DIGITAL_OCEAN = "data_do"
+
+SERVICE_PARAM_DOMAIN_NAME = "domain"
+SERVICE_PARAM_RECORD_NAME = "record"
+SERVICE_PARAM_RECORD_VALUE = "value"
+SERVICE_PARAM_RECORD_TYPE = "type"
```

Comment:
```suggestion
CONF_DOMAIN = "domain"
CONF_RECORD = "record"
CONF_VALUE = "value"
CONF_TYPE = "type"
```

Reply:
Does this really make a difference?

Reply:
We use `ATTR_*` for service call attributes:
```suggestion
ATTR_DOMAIN = "domain"
ATTR_RECORD = "record"
ATTR_VALUE = "value"
ATTR_TYPE = "type"
```